src/utils/api.js

```diff
@@ -16,6 +16,7 @@ const api = {
   containers: {
     get: id => get(`/containers/${id}`),
     list: () => get("/containers/list_root"),
+    listExport: () => get(`/containers/list_export/`, {format: 'csv'}),
```

Comment:
Inconsistent usage of backticks and double-quotes in the additions here.

Reply:
Comment here not addressed. The lines below use double-quote and this one uses backticks. It should be either one style or the other. Use double quotes if you don't need the string interpolation.

Reply:
I'll also add: `'csv'` should use double-ticks, for consistency with the other strings.